app/commands/generate_dist_command.rb

```diff
@@ -112,6 +112,8 @@ def generate_mapping_files(locale)
 
   def generate_mapping_group_files(locale, records)
     directory_path = "dist/#{locale}/integrations/#{records.first.integration.name}"
+    FileUtils.rm_rf(Dir.glob("#{directory_path}/*")) if directory_path.present?
```

Comment:
Should this be part of `make clean` or is it right for this to be part of _dist_?

If we keep this, can we find a way to shift parts of this into `System`. Trying to keep all file ops wrapped by that class

Reply:
+1: this is a good utility to have in `System`

Reply:
Done